packages/apputils/src/dialog.tsx

```diff
@@ -867,6 +867,7 @@ export namespace Dialog {
                   iconClass="jp-Icon"
                   className="jp-ToolbarButtonComponent-icon"
                   tag="span"
+                  title={trans.__('Close Icon, closes toolbar')}
```

Comment:
I usually recommend against words like icon, image, or the like because it is often otherwise declared or not more important than the fact it is an interactive area with an attached action. 

```suggestion
                  title={trans.__('Close toolbar')}
```